src/Symfony/Component/Workflow/Event/GuardEvent.php

```diff
@@ -41,8 +41,15 @@ public function isBlocked()
         return !$this->transitionBlockerList->isEmpty();
     }
 
+    /**
+     * @deprecated Deprecated since Symfony 5.0, use addTransitionBlocker instead.
+     *
+     * @param bool $blocked
+     */
     public function setBlocked($blocked)
     {
+        @trigger_error(sprintf('The "%s" function is deprecated since Symfony 5.0, use "%s" instead.', 'setBlocked', 'addTransitionBlocker'), E_USER_DEPRECATED);
```

Comment:
Since 4.4

Reply:
Is there a reason to process the string with sprintf and not write directly?